### ISSUE TITLE
The publish extensions github action is failing

### ISSUE THREAD

**Issue by joaosa** [open-vsx/publish-extensions#273]
It seems like it has been failing for a while now, as it can be verified [here](https://github.com/open-vsx/publish-extensions/actions?page=1&query=workflow%3A%22Publish+extensions+to+open-vsx.org%22)

There are a number of errors and they seem to be related to the following extensions:

* <https://github.com/arjun-g/vs-swagger-viewer>
```
> tsc -p ./

Error: src/preview/client.ts(101,30): error TS2339: Property 'swagger' does not exist on type 'string | object'.
  Property 'swagger' does not exist on type 'string'.
Error: src/preview/client.ts(104,28): error TS2339: Property 'openapi' does not exist on type 'string | object'.
  Property 'openapi' does not exist on type 'string'.
Error: src/preview/client.ts(105,28): error TS2339: Property 'openapi' does not exist on type 'string | object'.
  Property 'openapi' does not exist on type 'string'.
```

* <https://github.com/appulate/vscode-file-watcher>
```
> tsc -p ./

Error: src/extension.ts(139,10): error TS2322: Type 'readonly WorkspaceFolder[] | undefined' is not assignable to type 'WorkspaceFolder[] | undefined'.
```

* <https://github.com/alexkrechik/VSCucumberAutoComplete>
```
$ tsc -p ./
Error: node_modules/@types/node/index.d.ts(20,1): error TS1084: Invalid 'reference' directive syntax.
```

Is there anything I can do to help address this? I would probably start with checking the "faulty" extensions and seeing seeing if I can reproduce the errors :)

**Comment by spoenemann** [comment]
The GH action fails if publishing any extension fails, so to address this we'd need to fix all extensions :grin:. But the workflow still processes all extensions, so one failing extension should not affect the other extensions.

Sure, it would help a lot if you can contribute fixes to properly publish some more extensions. There are already lots of open issues about individual extensions failing to publish.

**Comment by joaosa** [comment]
Thanks for clarifying 😄 . I'll see what I can do regarding this!

As for the fix you applied yesterday, it seems the extensions still didn't get published 😅

**Comment by spoenemann** [comment]
Right, the publishing CLI detected an extension with missing license and asked for user input, but no input is available in CI 😅 (see the end of the [console log](https://github.com/open-vsx/publish-extensions/runs/1766657732?check_suite_focus=true)).

I created https://github.com/eclipse/openvsx/issues/240 to improve this and removed that extension from the list.

**Comment by GitMensch** [comment]
Even when the main issues here are now solved (like skipping the CI asking for a license) this seems like a general useful hint.
Maybe pin that, possibly renamed to "Why is the publish extensions github action failing"?
